Readme.md

```diff
@@ -162,6 +164,21 @@ If you have two-factor authentication (2FA) enabled on your account, you need to
 
 `$ docker run -d -e TZ=America/Chicago -e EMAIL=example@gmail.com -e PASSWORD=abc123 -e TOTP=ABC123 -e RUN_ON_STARTUP=true -e BASE_URL=https://example.com -e SMTP_HOST=smtp.gmail.com -e SMTP_PORT=587 -e SMTP_HOST=smtp.gmail.com -e EMAIL_SENDER_ADDRESS=hello@gmail.com -e EMAIL_SENDER_NAME="Epic Games Captchas" -e EMAIL_RECIPIENT_ADDRESS=hello@gmail.com -e SMTP_SECURE=true -e SMTP_USERNAME=hello@gmail.com -e SMTP_PASSWORD=abc123 -v /my/host/dir/:/usr/app/config:rw -p 3001:3000 charlocharlie/epicgames-freegames:latest`
 
+## Local Congifuration
+
+If for some reason you don't want to use Docker to run this tool you can run it from source by cloning this repo and installing [Node.js](https://nodejs.org/).
+
+1. Get this repo from Github
+    * Clone using git (recommended): `git clone https://github.com/claabs/epicgames-freegames-node.git`
+    * Or download and unpack ZIP archive: [epicgames-freegames-node](https://github.com/claabs/epicgames-freegames-node/archive/master.zip)
+1. Create `config` folder in the cloned/unpacked directory
+1. Create [JSON configuration](#json-configuration)
+1. Install Node.js
+1. Install Node.js dependecies
+    * Start terminal and navigate to cloned/unpacked directory
+    * Run `node ci`
+1. Start application: `./node_modules/.bin/ts-node src/index.ts`

```

Comment:
This is just the same as `npm run start:ts`. Is it expected that `npm` won't be available in the CLI this setup?

Reply:
`npm` comes with Node.js so it's available even on Windows. The reason I didn't use it here because it's:

1. A bit slower, makes ~1-2s difference on my machine
2. Some `npm` scripts in `package.json` are not Windows compatible (`npm run build` for example will fail since it uses `rm -rf`).

But I'm fine with replacing this with `npm run start:ts` since it should work everywhere, so it's up to you to decide.